Demo/Common/Networking/PaymentsRequest.swift

```diff
@@ -26,8 +26,9 @@ internal struct PaymentsRequest: Request {
     
     internal func encode(to encoder: Encoder) throws {
         var container = encoder.container(keyedBy: CodingKeys.self)
-        
+
         let currentConfiguration = ConfigurationConstants.current
+        let amount = data.amount ?? currentConfiguration.amount
```

Comment:
does this make sense? can there be a payment without an amount? if yes - no need have a fallback of `ConfigurationConstants.current.amount`. if no - no need for the instance variable to be optional

Reply:
payment is mandatory for a payment request